sale_automatic_workflow/__manifest__.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2011 Akretion Sébastien BEAU <sebastien.beau@akretion.com>
+# Copyright 2013 Camptocamp SA (author: Guewen Baconnier)
+# Copyright 2016 Sodexis
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Sale Automatic Workflow',
+    'version': '12.0.1.0.0',
+    'category': 'Sales Management',
+    'license': 'AGPL-3',
+    'author': "Akretion, "
+              "Camptocamp, "
+              "Sodexis, "
+              "Odoo Community Association (OCA)",
+    'website': 'https://github.com/OCA/sale-workflow',
+    'depends': [
+        'sale_stock',
+        'sales_team',
+    ],
+    'data': [
```

Comment:
Sequence as per [this](https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L27)